### DISCUSSION TITLE
Quadro / Fermi architecture support?

### DISCUSSION THREAD

**Discussion by tortoisedoc** [NVIDIA/open-gpu-kernel-modules#540] [Ideas]
IIRC, the cards in question are pretty outdated; would there be a chance to see them included in the list of supported GPU's?

Thanks!

**Comment by aritger** [comment]
I'm sorry: we only plan to support Turing and new GPU architectures with open-gpu-kernel-modules: the open kernel modules rely heavily on the "GSP" microprocessor on the GPU, which is only present on Turing and later.  Sorry.

**Reply by tortoisedoc** [reply]
Thank you for the clear response!
